samplesheets/tests/test_views_api_taskflow.py

```diff
@@ -144,6 +235,644 @@ def test_post_created(self):
         self.assertEqual(response.status_code, 400)
 
 
+class TestIrodsRequestCreateAPIView(TestIrodsRequestAPIViewBase):
+    """Tests for IrodsRequestCreateAPIView"""
+
+    def test_create(self):
+        """Test post() in IrodsRequestCreateAPIView"""
+        self.assertEqual(IrodsDataRequest.objects.count(), 0)
+        self._assert_alert_count(CREATE_ALERT, self.user, 0)
+        self._assert_alert_count(CREATE_ALERT, self.user_delegate, 0)
+        url = reverse(
+            'samplesheets:api_irods_request_create',
+            kwargs={'project': self.project.sodar_uuid},
+        )
+        with self.login(self.user_contrib):
+            response = self.client.post(url, self.post_data)
+            self.assertEqual(response.status_code, 200)
+
+        obj = IrodsDataRequest.objects.first()
+        self.assertEqual(IrodsDataRequest.objects.count(), 1)
+        self.assertEqual(obj.path, self.path)
+        self.assertEqual(obj.description, 'bla')
+        self._assert_alert_count(CREATE_ALERT, self.user, 1)
+        self._assert_alert_count(CREATE_ALERT, self.user_delegate, 1)
```

Comment:
Maybe also check here that a contributor/guest did not receive any alert? Oh and as a general note, good work with these tests!